buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -355,6 +359,10 @@ def build_multiple_images(self,
         for proc in processes:
             proc.join()
 
+        if cleanup_dockerfile:
```

Comment:
Doesn't this get cleaned up when the temp_dir is cleaned up? I'm a little confused why this extra step is needed if it's truly a temp dir

Reply:
Good question. But I don't believe it gets deleted because it was created using tempfile.NamedTemporaryFile(delete=False).  I did it this way to reuse existing code that was in the builder.py file. But I am open to doing another approach.

Reply:
If we don't delete the file explicitly when would the temp_dir get cleaned up?